lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -1119,10 +1124,12 @@ def test_dataset_sort(self, server_config, query_as, sort, results):
         # Assign "sequence numbers" in the inverse order of name
         test = User.query(username="test")
         all = Database.db_session.query(Dataset).order_by(desc(Dataset.name)).all()
+        mover = 1
         for i, d in enumerate(all):
             odd = i & 1
             Metadata.setvalue(d, "global.test.sequence", i)
-            Metadata.setvalue(d, "global.test.mcguffin", 100 - i)
+            Metadata.setvalue(d, "global.test.mcguffin", str(mover))
+            mover += 8
```

Comment:
Since you already have `i`, you don't really need `mover` -- it's just `i*8 + 1`.

Reply:
True, but not a big deal.